config/filter.d/nginx-http-errors.conf

```diff
@@ -0,0 +1,7 @@
+# Nginx client and server errors (HTTP 4xx or 5xx)
+#
+# Author: David Cook
+# Inspired by https://serverfault.com/a/849859/959055
+
+[Definition]
+failregex = <HOST> -.*"(GET|POST|HEAD).*HTTP.*" [4-5][0-9][0-9]
```

Comment:
Sorry, but no...
This regex is vulnerable (unanchored, has several catch-alls, etc)
The response code range 400-599 is too heavy by default, e. g. at least 401 (and probably 404) shall be handled differently.

Something like this possibly:
```ini
[Definition]
requri = /\S*
rescode = (?!40[14])[45]\d+
failregex = ^<ADDR> [^"]*"[A-Z]+\s+%(requri)s\s+[^"]*" %(rescode)s
```